pdfannots.py

```diff
@@ -414,7 +414,12 @@ def process_file(fh, codec, emit_progress):
                 if isinstance(a, pdftypes.PDFObjRef):
                     pdfannots.append(a.resolve())
                 else:
-                    sys.stderr.write('Warning: unknown annotation: %s\n' % a)
+                    sname = a['Subtype'].name
```

Comment:
Since you're just silencing a warning here, and you're right that we may want to handle this for some PDFs, I'll just leave the warning in place as-is.